clrs/_src/clrs_text/huggingface_generators_test.py

```diff
@@ -0,0 +1,141 @@
+"""Tests for clrs._src.clrs_text.huggingface_generators."""
+
```

Comment:
I believe that we still need to make tests clear (e.g. test a different behaviours in a separate tests) and add a regex match instead of beginning / end string match. I can write a commit updating the tests with how I see it and let you review it. How does this sound?

Reply:
Sounds good, I will await your reply on the other review questions before making an updated push and we can begin this process. In these current tests I was trying to emulate `clrs/clrs/_src/clrs_text/clrs_utils_test.py` to be sure the formatting is being preserved in the huggingface dataset.

Reply:
Oh, I see, apologies then. I will double check with the original author of that file why it was written in this way. Anyway, I think that it will be easier for everyone if we handle tests on our side, thank you!